net-libs/libaccounts_glib/libaccounts_glib-1.27.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="Accounts management library for GLib applications"
+DESCRIPTION="This project is a library for managing accounts which can be used from GLib \
+applications. It is part of the [accounts-sso project][accounts-sso]."
```

Comment:
This is badly formatted (the things in brackets).

Reply:
Granted :)